test/Microsoft.ML.CpuMath.UnitTests/UnitTests.cs

```diff
@@ -23,13 +24,14 @@ public class CpuMathUtilsUnitTests : BaseTestClass
         private static readonly FloatEqualityComparer _comparer;
         private static readonly FloatEqualityComparerForMatMul _matMulComparer;
         private static readonly string _defaultMode = "defaultMode";
-#if NETCOREAPP3_1
         private static Dictionary<string, string> _disableAvxEnvironmentVariables;
         private static Dictionary<string, string> _disableAvxAndSseEnvironmentVariables;
         private static readonly string _disableAvx = "COMPlus_EnableAVX";
         private static readonly string _disableSse = "COMPlus_EnableSSE";
         private static readonly string _disableAvxAndSse = "COMPlus_EnableHWIntrinsic";
-#endif
+        public static bool IsNetCore => Environment.Version.Major >= 5 || RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase);
```

Comment:
This property is mis-named. It shouldn't be "IsNetCore" when we are checking the version is `>= 5`.

"IsNet5OrGreater" / "IsNet5OrNewer"

Reply:
That line of code was taken directly from https://github.com/dotnet/runtime/blob/2be4fdc81f7f190ac35646ccc18799deb05da7dd/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L24. It still catches all of NetCore because of the `FrameWork` Description.

Reply:
In dotnet/runtime, we have different requirements - we don't run tests against earlier .NET Core versions right now. So the name may make sense there, but it doesn't here, where we run tests against earlier .NET Core versions.

Reply:
It does correctly catch all version of .Net core though, including .Net core 3. `RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase);` returns true for .Net core 3.

Reply:
Ah, ok. I think I read this as `Environment.Version.Major >= 5 && RuntimeInformation...` at first. I get it now.